docs/macos-server-setup.md

```diff
@@ -0,0 +1,92 @@
+# macOS Server Setup
+
+| Installation Guide | |
+| :- | :- |
+| This article is a part of the Installation Guide. You can read it alone or click on the previous link to easily move between the steps. |
+| [<< Step 2: Core Installation](core-installation.md) | [Step 4: Database Installation >>](database-installation.md) |
+
+## Extracting DBC, Maps, VMaps & MMaps
+
+Go to your AzerothCore build directory (e.g. $HOME/build/bin/) and copy the following files to your World of Warcraft binaries directory.
```

Comment:
isn't this duplicating the linux one?

Reply:
No, from the page I took this, written by Helias there are minor things different further down.